scripts/tests/test_validate_docstrings.py

```diff
@@ -455,6 +455,39 @@ def blank_lines(self, kind):
         """
         pass
 
+    def integer_parameter(self, kind):
+        """
+        Uses integer instead of int.
+
+        Parameters
+        ----------
+        kind : integer
+            Foo bar baz.
+        """
+        pass
+
+    def string_parameter(self, kind):
+        """
+        Uses string instead of str.
+
+        Parameters
+        ----------
+        kind : string
+            Foo bar baz.
+        """
+        pass
+
+    def boolean_parameter(self, kind):
+        """
+        Uses boolean instead of bool.
+
+        Parameters
+        ----------
+        kind : boolean
+            Foo bar baz.
+        """
```

Comment:
Can you add one test or two for something like `list of boolean`, `integer, float or string`... I think it makes sense to make sure that we capture them, and that the error in this case makes sense.